test/interface-tests/src/files/cp.js

```diff
@@ -368,45 +199,6 @@ export function testCp (factory, options) {
         ipfs = ipfsd.api
       })
 
-      it('copies a sharded directory to a normal directory', async () => {
-        const shardedDirPath = await createShardedDirectory(ipfs)
-
-        const normalDir = `dir-${Math.random()}`
-        const normalDirPath = `/${normalDir}`
-
-        await ipfs.files.mkdir(normalDirPath)
-
-        await ipfs.files.cp(shardedDirPath, normalDirPath)
-
-        const finalShardedDirPath = `${normalDirPath}${shardedDirPath}`
-
-        // should still be a sharded directory
-        await expect(isShardAtPath(finalShardedDirPath, ipfs)).to.eventually.be.true()
-        expect((await ipfs.files.stat(finalShardedDirPath)).type).to.equal('directory')
-
-        const files = await all(ipfs.files.ls(finalShardedDirPath))
-
-        expect(files.length).to.be.ok()
-      })
-
-      it('copies a normal directory to a sharded directory', async () => {
-        const shardedDirPath = await createShardedDirectory(ipfs)
-
-        const normalDir = `dir-${Math.random()}`
-        const normalDirPath = `/${normalDir}`
-
-        await ipfs.files.mkdir(normalDirPath)
-
-        await ipfs.files.cp(normalDirPath, shardedDirPath)
-
-        const finalDirPath = `${shardedDirPath}${normalDirPath}`
-
-        // should still be a sharded directory
-        await expect(isShardAtPath(shardedDirPath, ipfs)).to.eventually.be.true()
-        expect((await ipfs.files.stat(shardedDirPath)).type).to.equal('directory')
-        expect((await ipfs.files.stat(finalDirPath)).type).to.equal('directory')
-      })
```

Comment:
Maybe I missed it, but what's the reason for so many sharding-copy tests to be removed?

Reply:
they were all "not implemented in kubo" or "not supported by go-ipfs" see https://github.com/ipfs/js-ipfs/blob/74aee8b3d78f233c3199a3e9a6c0ac628a31a433/packages/ipfs/test/interface-http-go.js#L392 for this particular test

Reply:
fwiw [autosharding is implemented since Kubo 0.11](https://github.com/ipfs/kubo/blob/master/docs/changelogs/v0.11.md#-automatic-unixfs-sharding), but i am ok with removing them: having sharding tests in HTTP client makes little sense – this was more an interop test between GO and JS then the test of HTTP client.